content/en/tracing/live_debugger/_index.md

```diff
@@ -0,0 +1,164 @@
+  tag: "Documentation"
+  text: "Dynamic Instrumentation Expression Language"
+- link: "/dynamic_instrumentation/sensitive-data-scrubbing/"
+  tag: "Documentation"
+  text: "Sensitive Data Scrubbing"
+- link: "/dynamic_instrumentation/symdb/"
+  tag: "Documentation"
+  text: "Autocomplete and Search (Preview)"
+- link: "/tracing/error_tracking/execution_replay"
+  tag: "Documentation"
+  text: "Execution Replay"
+---
+
+{{< beta-callout-private url="https://www.datadoghq.com/product-preview/live-debugger/" >}}
+    Live Debugger is in Limited Preview. Request access to join the waiting list.
+    <br>
+    To submit questions, feedback, or requests related to Live Debugger, fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdM9SV4fxrM_OvQ2CtI7CMl7evN0jasFb6X1QiPAbW6dPTQVQ/viewform?usp=header">this form</a> with details. 
+    <br>
+    For urgent issues, contact <a href="https://www.datadoghq.com/support/">Datadog support</a>.
+{{< /beta-callout-private >}}
+
+## Overview
+
+With Live Debugger, you can debug running applications in real time, without redeploying code or interrupting service. Powered by Datadog's [Dynamic Instrumentation][1], Live Debugger uses logpoints&mdash;auto-expiring, "non-breaking breakpoints"&mdash;to collect information from running applications without pausing execution. This makes it ideal for investigating issues in environments where traditional debugging methods aren't practical.
+
+## Key capabilities
+
+Live Debugger provides:
+
+- **Real-time inspection** of variable states, method arguments, and execution paths in running code.
+- **Non-invasive data collection** that captures debugging information without stopping applications or degrading performance.
+- **Code instrumentation** with logpoints that can be added anywhere in your code, including third-party libraries.
+- **Auto-expiring logpoints** that are automatically deactivated after a set time (default: 48 hours).
+- **Conditional logging** based on user-defined criteria to capture data only when specific conditions are met.
+- **Built-in [sensitive data scrubbing][3]** to prevent exposure of personal information, passwords, and secrets.
+
+## Getting started
+
+### Prerequisites
+
+1. All [Dynamic Instrumentation prerequisites][16] are met.
+2. (For one-click enablement) You have the following permissions: **Org Management**, **APM Remote Configuration Read**, **APM Remote Configuration Write**.
+3. You have [created a logs index][19] to store debugging information. 
+4. (Recommended) You have enabled [Source Code Integration][20] to view and select specific code locations when adding logpoints.
+
+### Setup Live Debugger
+
+Enable and disable Live Debugger on a service using one of the following methods:
+
+#### One-click enablement (recommended) ####
+
+<div class="alert alert-info">Only users with eligible permissions can use one-click enablement. Users may need to re-authenticate before using this feature.</div>
```

Comment:
1. As above, recommending that we move the blurb above about which permissions are needed here. 
2. Mentioning the need for re-authentication as a note above makes it sound like a warning/deterrent and might scare folks off, perhaps it's better to include it in the numbered list below instead?